test/createModel.spec.js

```diff
@@ -0,0 +1,45 @@
+/* global describe, it */
+import chai, { expect } from 'chai';
+import sinon from 'sinon';
+import sinonChai from 'sinon-chai';
+
+import BaseModel from '../src/Model';
+import createModel from '../src/createModel';
+chai.use(sinonChai);
```

Comment:
Agree, but then again, you don't need to load it all the time :) This is a case ;-)